src/DraxList.tsx

```diff
@@ -210,7 +211,7 @@ export const DraxList = <T extends unknown>(
 			} = itemStyles ?? {};
 			return (
 				<DraxView
-					style={[itemStyle, { transform: getShiftTransform(originalIndex) }]}
+					style={[itemStyle, getShiftTransformStyle(originalIndex)]}
```

Comment:
We may want to tweak this to merge transform styles using the function I added when addressing #35

Reply:
Looks like you mean this commit (https://github.com/nuclearpasta/react-native-drax/pull/88/files) I probably need to talk this one out, can't wrap my head around how to use it.